README.md

```diff
@@ -75,8 +75,128 @@ Moltin::Client.new({
 })
 ```
 
+### Currency
+
+Coming Soon.
+
 ## Usage
 
+The client used in the following code examples has been created like this:
+
+```
+client = Moltin::Client.new({
+  client_id: 'YOUR_CLIENT_ID',
+  client_secret: 'YOUR_CLIENT_SECRET'
+})
+```
+
+### Resources (Products, more coming soon)
+
+#### Retrieve a list of resources (GET)
+
+##### Products
+
+```
+client.products.all
+# => [Moltin::Models::Product, Moltin::Models::Product]
+```
+
+##### Customizing the requests
+
+Coming Soon.
+
+###### Pagination
+
+Coming Soon.
+
+###### Sorting
+
+Coming Soon.
+
+###### Filtering
+
+Coming Soon.
+
+###### Including data
+
+Coming Soon.
+
+##### Specific cases
+
+###### Fetching the category/brand/collection tree
+
+Coming Soon.
+
+#### Retrieve a single resource (GET)
+
+##### Products
+
+```
+client.products.find(product_id)
```

Comment:
Should reflect the JS/PHP SDK with "Get" like so:

```
Moltin.Products.Get('productId').then((product) => {
  // Do something
});

$moltin->products->get($productID);
```

Instead of find